x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps/install_cloud_formation_managed_agent_step.tsx

```diff
@@ -21,15 +23,22 @@ export const InstallCloudFormationManagedAgentStep = ({
   enrollToken,
   isComplete,
   cloudFormationTemplateUrl,
+  agentPolicy,
 }: {
   selectedApiKeyId?: string;
   apiKeyData?: GetOneEnrollmentAPIKeyResponse | null;
   enrollToken?: string;
   isComplete?: boolean;
   cloudFormationTemplateUrl: string;
+  agentPolicy?: AgentPolicy;
 }): EuiContainedStepProps => {
   const nonCompleteStatus = selectedApiKeyId ? undefined : 'disabled';
   const status = isComplete ? 'complete' : nonCompleteStatus;
+
+  const cloudSecurityPackagePolicy = agentPolicy?.package_policies?.find(
+    (p) => p.package?.name === 'cloud_security_posture'
```

Comment:
prefer using the constant for the `cloud_security_posture` value that's in `fleet/common`